pkg/util/kube/utils.go

```diff
@@ -187,6 +194,27 @@ func isProvisionedByCSI(log logrus.FieldLogger, pv *corev1api.PersistentVolume,
 	return false, nil
 }
 
+func VerifyVolumePath(path string, fs filesystem.Interface) (string, error) {
```

Comment:
I think adding the moutnpoint check makes sense.  
However, it makes the successful UT fail early which may cover some real code errors in the future.
Additionally, this is an exported function and is not self-explanatory as `singlePathMatch`, so please add some comments to explain what it does and in what cases this should be called.

Reply:
Tested with emptyDir volume. It is not a mount point, so not all volume types are mount points in pods.
Removed mount point checking.